### DISCUSSION TITLE
[Question] MMKV 在 WIN32 上动态库和静态库有区别吗？

### DISCUSSION THREAD

**Discussion by zh1an** [Tencent/MMKV#766] [Q&A] [answered]
现在 `MMKV` 在 `WIN32` 上默认是编译成静态库，我尝试将其编译成动态库，并且成功了。在使用过程中，动态库和静态库的使用会有什么区别吗？动态库的性能会不会跟静态库的不一致？为什么官方是默认使用静态库的呢？

PS：
本人的程序主要是用于多进程间存取数据。

**Comment by lingol** [comment, marked as answer]
No, not much difference. Use whatever suits you best.
